### PR TITLE
Fix variable names and doc wording

### PR DESCRIPTION
**Pull request by ayan4m1**
This is not exhaustive, but a step in the direction of conforming all source to use the [conventions](https://msdn.microsoft.com/en-us/library/ms229045(v=vs.110).aspx) that were [suggested](https://msdn.microsoft.com/en-us/library/ms229002(v=vs.110).aspx) by Microsoft.

### CODE REVIEW

File: Core/Types/Kraken.cs [hunk 1]
```diff
@@ -265,7 +265,7 @@ public override string Message
         // our base class?
 
         public ModNotInstalledKraken(string mod, string reason = null, Exception inner_exception = null)
-            :base(reason, inner_exception)
+            : base(reason, inner_exception)
```
Comment (politas) [inline Core/Types/Kraken.cs]: Shouldn't this be innerException, too?

**Comment by politas** [comment]
I don't think the forum thread link in the Contribution guidelines page has been updated to the new thread.

**Comment by ayan4m1** [comment]
Rebased and updated

File: GUI/Main.Designer.cs [hunk 2]
```diff
@@ -481,7 +476,7 @@ private void InitializeComponent()
             this.ModList.ColumnHeadersHeightSizeMode = System.Windows.Forms.DataGridViewColumnHeadersHeightSizeMode.AutoSize;
             this.ModList.Columns.AddRange(new System.Windows.Forms.DataGridViewColumn[] {
             this.Installed,
-            this.UpdateCol,
+            this.Update,
```
Comment (politas) [inline GUI/Main.Designer.cs]: Please revert, or expand name to UpdateColumn or UpdateSelection or some such. "Update" gives us build warnings.

File: GUI/Main.Designer.cs [hunk 3]
```diff
@@ -513,10 +509,10 @@ private void InitializeComponent()
...
             // 
-            this.UpdateCol.HeaderText = "Update";
-            this.UpdateCol.Name = "Update";
-            this.UpdateCol.SortMode = System.Windows.Forms.DataGridViewColumnSortMode.Programmatic;
-            this.UpdateCol.Width = 46;
+            this.Update.HeaderText = "Update";
+            this.Update.Name = "Update";
+            this.Update.SortMode = System.Windows.Forms.DataGridViewColumnSortMode.Programmatic;
+            this.Update.Width = 46;
             //
```
Comment (politas) [inline GUI/Main.Designer.cs]: Please revert, or expand name to UpdateColumn or UpdateSelection or some such. "Update" gives us build warnings.

File: GUI/Main.Designer.cs [hunk 4]
```diff
@@ -64,7 +65,7 @@ private void InitializeComponent()
             this.splitContainer1 = new System.Windows.Forms.SplitContainer();
             this.ModList = new CKAN.MainModListGUI();
             this.Installed = new System.Windows.Forms.DataGridViewCheckBoxColumn();
-            this.UpdateCol = new System.Windows.Forms.DataGridViewCheckBoxColumn();
+            this.Update = new System.Windows.Forms.DataGridViewCheckBoxColumn();
```
Comment (politas) [inline GUI/Main.Designer.cs]: Please revert, or expand name to UpdateColumn or UpdateSelection or some such. "Update" gives us build warnings.

    Main.Designer.cs(1614,44): warning CS0108: `CKAN.Main.Update' hides inherited member `System.Windows.Forms.Control.Update()'. Use the new keyword if hiding was intended
    /usr/lib/mono/4.5-api/System.Windows.Forms.dll (Location of the symbol related to previous warning)
    Main.Designer.cs(1600,44): warning CS0169: The private field `CKAN.Main.UpdateCol' is never used

File: GUI/MainInstall.cs [hunk 5]
```diff
@@ -227,20 +227,20 @@ private static bool WasSuccessful(Action action)
...
-                if (ex.owning_module != null)
+                if (ex.owningModule != null)
                 {
                     GUI.user.RaiseMessage(
                         "\r\nOh no! We tried to overwrite a file owned by another mod!\r\n" +
                         "Please try a `ckan update` and try again.\r\n\r\n" +
                         "If this problem re-occurs, then it maybe a packaging bug.\r\n" +
                         "Please report it at:\r\n\r\n" +
-                        "https://github.com/KSP-CKAN/CKAN/issues/new\r\n\r\n" +
+                        "https://github.com/KSP-CKAN/CKAN-meta/issues/new\n\n" +
```
Comment (politas) [inline GUI/MainInstall.cs]: "https://github.com/KSP-CKAN/NetKAN/issues/new\r\n\r\n" +

File: Core/Types/Version.cs [hunk 6]
```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Text.RegularExpressions;
 using Newtonsoft.Json;
+using System.Reflection;
```
Comment (politas) [inline Core/Types/Version.cs]: Why this?

**Comment by politas** [comment]
Whew! That's a lot to review!

File: GUI/Main.Designer.cs (hunk 4 above)
Comment (ayan4m1) [inline GUI/Main.Designer.cs]: OK, it looks like there were two declarations in the Designer file... I have had some issues mixing VS and the massive Main designer, will be more aware of them in the future, thanks.

Comment (politas) [inline GUI/Main.Designer.cs]: It's not internal duplication, it's a conflict with System.Windows.Forms.dll

File: Core/Types/Version.cs (hunk 6 above)
Comment (ayan4m1) [inline Core/Types/Version.cs]: Experimented with another method and forgot to install Resharper on this machine, so VS didn't auto-remove and sort usings.

**Comment by ayan4m1** [comment]
Yep, and thanks for doing so! I made the changes related to points you brought up.

**Pull request merged**